modin/engines/ray/pandas_on_ray/frame/axis_partition.py

```diff
@@ -83,4 +97,10 @@ def deploy_ray_func(func, *args):  # pragma: no cover
     Returns:
         The result of the function `func`.
     """
-    return func(*args)
+    result = func(*args)
+    if isinstance(result, pandas.DataFrame):
+        return result, len(result), len(result.columns)
```

Comment:
We should add this change to the python backend as well?

Reply:
Possibly, let me look into it

Reply:
The conclusion I reached is that this is a Ray-specific optimization, so we don't need to add it to the Python backend.